berkeley-mobile/Map/MapViewController.swift

```diff
@@ -99,14 +104,44 @@ class MapViewController: UIViewController, SearchDrawerViewDelegate {
     /// Repoisitions the map's compass so that it is not obscured by the search bar.
     private func updateCompassPosition() {
         mapView.showsCompass = false
-        let compass = MKCompassButton(mapView: mapView)
+        compass = MKCompassButton(mapView: mapView)
         view.addSubview(compass)
         // Position the compass to bottom-right of `FilterView`
         compass.translatesAutoresizingMaskIntoConstraints = false
         compass.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor).isActive = true
         compass.topAnchor.constraint(equalTo: filterView.bottomAnchor, constant: kViewMargin).isActive = true
     }
     
+    /// Sets up the button that jumps to user location
+    private func setUpUserLocationButton() {
+        let _buttonWidth = mapView.frame.width * 0.12
+        userLocationButton = UIButton(type: .custom)
+        view.addSubview(userLocationButton)
+        userLocationButton.translatesAutoresizingMaskIntoConstraints = false
+        userLocationButton.backgroundColor = UIColor.white
+        userLocationButton.widthAnchor.constraint(equalToConstant: _buttonWidth).isActive = true
+        userLocationButton.heightAnchor.constraint(equalToConstant: _buttonWidth).isActive = true
+        userLocationButton.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor).isActive = true
+        userLocationButton.topAnchor.constraint(equalTo: filterView.bottomAnchor, constant: kViewMargin).isActive = true
+        userLocationButton.clipsToBounds = true
+        userLocationButton.layer.masksToBounds = false
+        userLocationButton.layer.cornerRadius = mapView.frame.width * 0.12 * 0.5
+        userLocationButton.layer.shadowRadius = 5
+        userLocationButton.layer.shadowOpacity = 0.2
+        userLocationButton.layer.shadowColor = UIColor.black.cgColor
+        userLocationButton.layer.shadowOffset = CGSize(width: 0, height: 10)
```

Comment:
shadow looks a bit different from FilterViewCell but I might just be seeing wrong. want to make sure the shadow stays consistent